.env.example

```
@@ -59,6 +59,13 @@ DART_API_KEY=
 # FORK: ECOS (Bank of Korea economic statistics) — free key at https://ecos.bok.or.kr/api/
 ECOS_API_KEY=
 
+# FORK: Qdrant Cloud — 한국 공시 벡터 DB (RAG 파이프라인)
+# 가입: https://cloud.qdrant.io (free tier 1GB, 도쿄 리전 권장)
+# 클러스터 생성 후 대시보드에서 엔드포인트 URL 과 API Key 복사
+# 로컬 개발 시엔 비워두고 Docker Compose 의 qdrant 서비스 사용 가능
+QDRANT_URL=
+QDRANT_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QDRANT_API_KEY key should go before the QDRANT_URL key

(UnorderedKey)